models/users.js

```diff
@@ -249,6 +255,23 @@ const fetchUser = async ({ userId = null, username = null, githubUsername = null
         id = doc.id;
         userData = doc.data();
       });
+    } else if (userIds) {
+      const users = await userModel
+        .where(admin.firestore.FieldPath.documentId(), "in", userIds)
+        .where("roles.archived", "==", false)
```

Comment:
firestore IN has a limit of 30 values, I think 
So what will happen if there are 30+ userIds here?

Reply:
You are right, Firestore has a limit of 30 values for the 'in' operator. I have fixed this by fetching the data in smaller chunks now. Thanks for pointing that out.